### ISSUE TITLE
Unable to generate implants on latest versions

### ISSUE THREAD

**Issue by gettheguru** [BishopFox/sliver#172]
**Describe the bug**
When using release version 0.0.6 I have no issues. When building from source, or using releases from other repos (e.g. https://github.com/dzflack/sliver/releases) for latest version (0.0.7) I am unable to build implants.

**To Reproduce**
1. Clone source into gopath directory $GOPATH/src/github.com/bishopfox/sliver
2. Run ./go-assets.sh
3. Run 'make static-linux'
4. Execute ./sliver-server
5. generate --mtls 1.2.3.4

Error with obfuscation from the sliver.log
ERRO[0021] [sliver/server/gobfuscate/main.go:78] Failed to obfuscate package names: package move: couldn't load packages due to errors: github.com/bishopfox/
sliver/protobuf/sliver 

Error without obfuscation from the sliver.log:
INFO[0034] [sliver/server/db/db.go:168] No bucket for name "slivers" 
INFO[0034] [sliver/server/db/db.go:175] Created new bucket with name "slivers" (2f5e2024-59b3-42ff-aa2f-89b31d809719) 
DEBU[0034] [sliver/server/db/db.go:199] Loading db from /root/.sliver/db/buckets/2f5e2024-59b3-42ff-aa2f-89b31d809719 
INFO[0034] [/root/go-workspace/pkg/mod/github.com/dgraph-io/badger@v1.6.1/logger.go:46] All 0 tables opened in 0s 
INFO[0034] [sliver/server/generate/slivers.go:96] Saved config for 'AVAILABLE_MEET' 
ERRO[0034] [sliver/server/generate/binaries.go:361] Failed to save file to db open /root/.sliver/slivers/windows/amd64/AVAILABLE_MEET/obfuscated/bin/AVAILABLE_MEET.exe: no such file or directory %!s(<nil>) 

**Expected behavior**
Sliver should compile normally

**Screenshots**

![image](https://user-images.githubusercontent.com/46968858/79205473-e2b37680-7e35-11ea-98de-7874c5bf10f4.png)

I can manually move the protobuf packages into this .sliver directory from my $GOPATH and the error will dissapear about the missing packages **but the sliver creation will still fail with the same errors.**

**Desktop (please complete the following information):**
 - OS: Kali Linux 2020.1
root@kali:~/RastaLabs/C2-SLIVER# protoc --version
libprotoc 3.11.4

**Additional context**
The fact that the pre-compiled binary for 0.0.7 (from the other repo) also has the same issue makes me think this must be something to do with my specific environment rather than the compilation process. Please let me know if you need any further details.

**Comment by rkervella** [comment]
Alright, I'm able to reproduce on a fresh docker container. Definitely a bug.

**Comment by rkervella** [comment]
Here are the logs I get:
```
...
INFO[0078] [sliver/server/gogo/go.go:106] PATH=/root/.sliver/go/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin 
INFO[0078] [sliver/server/gogo/go.go:108] --- stdout ---
 
INFO[0078] [sliver/server/gogo/go.go:109] --- stderr ---
protobuf/sliver/constants.go:11:2: cannot find package "google.golang.org/protobuf/reflect/protoreflect" in any of:
        /root/.sliver/go/src/google.golang.org/protobuf/reflect/protoreflect (from $GOROOT)
        /root/.sliver/slivers/windows/amd64/SURPRISING_CHAIN/src/google.golang.org/protobuf/reflect/protoreflect (from $GOPATH)
protobuf/sliver/constants.go:12:2: cannot find package "google.golang.org/protobuf/runtime/protoimpl" in any of:
        /root/.sliver/go/src/google.golang.org/protobuf/runtime/protoimpl (from $GOROOT)
        /root/.sliver/slivers/windows/amd64/SURPRISING_CHAIN/src/google.golang.org/protobuf/runtime/protoimpl (from $GOPATH)
 ...
```

**Comment by moloch--** [comment]
Must be a breaking change in the `protobuf` dependency? I think it's the one thing we haven't locked to a specific version.

**Comment by rkervella** [comment]
Yeah I believe so. Also I just noticed this:
>This module (github.com/golang/protobuf) contains Go bindings for protocol buffers.
It has been superseded by the google.golang.org/protobuf module, which contains an updated and simplified API, support for protobuf reflection, and many other improvements. We recommend that new code use the google.golang.org/protobuf module.

From [the protobuf repo we use](https://github.com/protocolbuffers/protobuf-go).

**Comment by rkervella** [comment]
Looks like there has been a lot of updates: https://developers.google.com/protocol-buffers/docs/reference/go-generated

**Comment by moloch--** [comment]
This is implemented in the gRPC branch, but yea we probably need to backport it.

**Comment by gettheguru** [comment]
Great to see you've identified the cause. Is there something I can do to get it working for now?

**Comment by rkervella** [comment]
I'm not entirely sure, but it might be related to PR #169 
I guess it's worth testing by reverting to commit 34291e and re-running `./go-assets.sh`.

**Comment by rkervella** [comment]
Doesn't seem to work. Might be related to the latest protoc-go version mixed with the updated protobuf repo.

**Comment by moloch--** [comment]
I can take a look later today too; @gettheguru not sure i have a great short term solution for you other than to revert to an earlier version of `protoc` and `protoc-go`

**Comment by moloch--** [comment]
My local version of `master` still builds, I'll try to lock it to whatever my local version is.

**Comment by moloch--** [comment]
Fixed in https://github.com/BishopFox/sliver/commit/eb2bcd174705320e8c3dce19c3ea17b128333f54 --basically [Protobuf updated to v1.4.0 a couple days ago](https://github.com/golang/protobuf/releases/tag/v1.4.0), which included some breaking changes. Since `go get` cannot be used to specify a version and we can't use Go modules to install a specific version of `protoc-gen-go` we instead `wget` and manually build the specific version we need. I've also done the same with our other `go get` external dependency: `packr`, which now manually builds v1.30.1. I've also updated the wiki "Compile from Source" instructions to reflect this.
